specification/maps/data-plane/Creator/preview/2022-09-01-preview/features.json

```diff
@@ -0,0 +1,1074 @@
+          "type": "string",
+          "example": "3.0.1"
+        },
+        "info": {
+          "description": "The OpenAPI info object.",
+          "type": "object",
+          "properties": {
+            "title": {
+              "description": "The title of the API.",
+              "type": "string",
+              "example": "Azure Maps Web Feature Service"
+            },
+            "version": {
+              "description": "The version of the API.",
+              "type": "string",
+              "example": "2022-09-01-preview"
+            },
+            "description": {
+              "description": "The description of the API.",
+              "type": "string",
+              "example": "Azure Maps Web Feature REST APIs"
+            }
+          }
+        },
+        "paths": {
+          "description": "The OpenAPI paths object.",
+          "type": "object"
+        },
+        "components": {
+          "description": "The OpenAPI components object.",
+          "type": "object",
+          "properties": {
+            "parameters": {
+              "description": "The OpenAPI parameters object.",
+              "type": "object"
+            },
+            "schemas": {
+              "description": "The OpenAPI schemas object.",
+              "type": "object"
+            }
+          }
+        },
+        "tags": {
+          "description": "The OpenAPI tags object.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "description": "The name of the tag.",
+                "type": "string",
+                "example": "Features"
+              },
+              "description": {
+                "description": "The description of the tag.",
+                "type": "string",
+                "example": "Access to data (features)."
+              }
+            }
+          }
+        }
+      }
+    },
+    "ConformanceResult": {
+      "type": "object",
+      "required": [
+        "conformsTo"
+      ],
+      "properties": {
+        "conformsTo": {
+          "description": "List of requirements classes that the server conforms to.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "example": [
+            "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/core",
+            "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/oas3",
+            "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/geojson"
+          ]
+        }
+      }
+    },
+    "FeaturesEndpointLink": {
+      "description": "Links to other Features endpoints. For more details about link relations, please visit the [OGC API - Features - Part 1](http://www.opengis.net/doc/IS/ogcapi-features-1/1.0.1#_link_relations) standard.",
+      "type": "object",
+      "required": [
+        "href"
+      ],
+      "properties": {
+        "href": {
```

Comment:
This probably should be annotated with `"format": "uri"`.

Reply:
Good catch, updated ✔️